modules/document_repository/test/TestPlan

```diff
@@ -1,44 +1,46 @@
    [Manual Testing]
-5. Download a file and check content.
+6. Edit document property and save. Check that changes were saved properly.
    [Manual Testing]
-6. Add files in repository to test search according to file name, file type, version, site and uploaded by.
+7. Download a file and check content.
    [Manual Testing]
-7. Check that expand/collapse works for non-empty categories. Check that when performing a search, the categories
+8. Add files in repository to test search according to file name, file type, version, site and uploaded by.
+   [Manual Testing]
+9. Check that expand/collapse works for non-empty categories. Check that when performing a search, the categories
    containing the resulting files (and only those) are shown in a list format
    [Manual Testing]
-8. Check that if a category contains special characters (e.g. space, dot or comma) it can be expanded.
+10. Check that if a category contains special characters (e.g. space, dot or comma) it can be expanded.
    [Manual Testing]
-9. Add two files with the same name but with different contents in two different categories. Verify that, when 
+11. Add two files with the same name but with different contents in two different categories. Verify that, when 
    downloaded, their contents are different.
    [Manual Testing]
-10. Edit a file in the repository: check that “Last modified on” date is updated.
+12. Edit a file in the repository: check that “Last modified on” date is updated.
     [Manual Testing]
-11. Upload a file using another user and then update it. Check that “Last updated by” field is updated.
+13. Upload a file using another user and then update it. Check that “Last updated by” field is updated.
     [Manual Testing]
-12. User A edits a file. User B tries to delete the same file in the meantime. Check that he can't.
+14. User A edits a file. User B tries to delete the same file in the meantime. Check that he can't.
     [Manual Testing]
-13. Check that preference “Receive Document Repository Notifications” works.
+15. Check that preference “Receive Document Repository Notifications” works.
     [Manual Testing]
-14. Check that performance is OK with large number of files.
+16. Check that performance is OK with large number of files.
     [Manual Testing]
-15. Check that if you have the 'Receive document repository notification' activated you will receive an email each 
+17. Check that if you have the 'Receive document repository notification' activated you will receive an email each 
     time one of the following event occurs:
        - Addition, deletion or modification of a file
        - Addition of a category.
     Also check that the www address contained in the notification email is correct.
     [Manual Testing]
-16. Check that the following settings are in /etc/php5/apache2/php.ini:
+18. Check that the following settings are in /etc/php5/apache2/php.ini:
```

Comment:
this should probably be changed to just be `php.ini`
nobody should be running php < 7

Reply:
also maybe this should be put towards the beginning of the test plan?

Reply:
For the most part, I don't think the document repository expects large files (mostly documents, I'd assume, as the name suggests), so I've added additional text at the beginning of the last point for uploading large files. 
Because I think its more of an optional change, leaving it at the end seems better. What do you think?